src/Statistics.dfy

```diff
@@ -0,0 +1,50 @@
+include "../libraries/src/Collections/Sequences/Seq.dfy"
+include "../libraries/src/Collections/Maps/Maps.dfy"
+
+include "Externs.dfy"
+
+module Statistics {
+  import Externs
+  import Seq
+
+  function method Square(n: real): real { n * n }
+
+  function method Sum(xs: seq<real>): real {
+    Seq.FoldLeft((a, b) => a + b, 0.0, xs)
```

Comment:
We really should have a version of sum without the starting parameter.  Not in this PR though.

Reply:
Agreed.